AppinfoJsonFileType.js

```diff
@@ -34,6 +35,42 @@ var AppinfoJsonFileType = function(project) {
     this.pseudo = this.API.newTranslationSet(project.getSourceLocale());
 };
 
+var defaultMappings = {
+    "**/appinfo.json": {
+        "template": "[dir]/[resDir]/[locale]/[filename]"
```

Comment:
`[locale]` -> `[localeDir]`  because locale will come out like "ko-KR", whereas localeDir comes out as "ko" or "ko/KR" depending on the locale mapping

Reply:
which means, all the locales that we support from webOS  should be written in locale map list. 
currently we want to only  write  different case in locale map  because general normal case we can get from ilib in code 
something like es-MX:es. because es-ES is the es's localemap in iLib